react-accordion-component/my-app/src/Accordion.jsx

```diff
@@ -0,0 +1,41 @@
+import { useState } from 'react';
+
+export default function Accordion({ topics }) {
+  const [isActive, setIsActive] = useState(false);
+
+  function handleTopicClick(topicName) {
+    setIsActive(topicName === isActive ? true : topicName);
```

Comment:
Same as above, set to null if no topic is to be selected: 
`setIsActive(topicName === isActive ? null : topicName);